package.json

```diff
@@ -1,64 +1,25 @@
 {
   "name": "irving",
-  "version": "1.0.0",
+  "private": true,
   "description": "Irving is a ReactJS based, isomorphically rendered, headless CMS frontend application.",
-  "main": "main.js",
-  "scripts": {
-    "build": "BABEL_ENV=app lerna run build",
-    "preinstall": "npx check-node-version --package",
-    "install": "lerna bootstrap --hoist",
-    "link": "lerna link",
-    "test": "BABEL_ENV=test lerna run test --parallel",
-    "test:watch": "BABEL_ENV=test lerna run test:watch",
-    "styleguide:dev": "NODE_ENV=development npx styleguidist server --config ./config/styleguide.config.js",
-    "styleguide:build": "NODE_ENV=production npx styleguidist build --config ./config/styleguide.config.js"
-  },
+  "homepage": "https://github.com/alleyinteractive/irving#readme",
   "repository": {
     "type": "git",
     "url": "git@github.com:alleyinteractive/irving.git"
   },
-  "author": "Alley (Alley.co)",
   "license": "MIT",
-  "homepage": "https://github.com/alleyinteractive/irving#readme",
-  "engines": {
-    "node": "10",
-    "npm": "6"
+  "author": "Alley (Alley.co)",
+  "main": "main.js",
+  "scripts": {
+    "build": "BABEL_ENV=app lerna run build",
+    "preinstall": "npx check-node-version --package",
+    "install": "lerna bootstrap --hoist --nohoist=@irvingjs/*",
```

Comment:
this will prevent hoisting of internal packages (which I think is an lerna bug...)